src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1365,6 +1367,36 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly customDriverAnnotationReport = remoteData<{ hasBinary:boolean, tiers:string[] }>({
+        await:()=>[
+            this.mutations
+        ],
+        invoke:()=>{
+            let hasBinary = false;
+            let tiersMap:{[tier:string]:boolean} = {};
+            for (const mutation of this.mutations.result!) {
+                hasBinary = hasBinary || !!mutation.driverFilter;
+                mutation.driverTiersFilter && (tiersMap[mutation.driverTiersFilter] = true);
```

Comment:
i don't think we should use this syntax for normal control logic.   we use it in jsx for a specic typescript reason.  lets use if () {}

Reply:
one